lib/ast.js

```diff
@@ -345,7 +345,7 @@ var AST_Toplevel = DEFNODE("Toplevel", "globals", {
 var AST_Expansion = DEFNODE("Expansion", "expression", {
     $documentation: "An expandible argument, such as ...rest, a splat, such as [1,2,...all], or an expansion in a variable declaration, such as var [first, ...rest] = list",
     $propdoc: {
-        expression: "AST_Symbol the thing to be expanded"
+        expression: "[AST_Symbol] the thing to be expanded"
```

Comment:
Don't worry - I'm equally guilty in not reading what I was fixing. I literally only reacted to the formatting issue without even parsing the words themselves.

(Okay, may be I did channel my inner :paperclip: and perform _Spell Check_...)